opentelemetry-sdk/src/opentelemetry/sdk/_metrics/instrument.py

```diff
@@ -96,65 +71,69 @@ def __init__(
         unit: str = "",
         description: str = "",
     ):
-        super().__init__(
-            instrumentation_info,
-            name,
-            callback,
-            unit=unit,
-            description=description,
-        )
 
+        self._instrumentation_info = instrumentation_info
+        super().__init__(name, callback, unit=unit, description=description)
```

Comment:
It doesn't look like `callback` should be passed to any of the MRO constructors

Reply:
Yes, the corresponding API `__init__` method is not doing anything with any argument now, but I rather pass them still in case we decide on some API argument validation.